src/libpriv/rpmostree-core.c

```diff
@@ -1842,8 +1836,12 @@ rpmostree_dnf_add_checksum_goal (GChecksum   *checksum,
           if (!ostree_repo_resolve_rev (pkgcache, cachebranch, FALSE, &cached_rev, error))
             return FALSE;
 
+          g_autoptr(GVariant) commit = NULL;
+          if (!ostree_repo_load_commit (pkgcache, cachebranch, &commit, NULL, error))
```

Comment:
This should be `cached_rev`, right?

Reply:
Yes!  Good catch, I had been banging my head against that one for a bit and kept not seeing that.